modules/cas_cache/layer_cache_management.c

```diff
@@ -1868,10 +1861,10 @@ int cache_mngt_init_instance(struct ocf_mngt_cache_config *cfg,
 
 	context->rollback_thread = kthread_create(cache_start_rollback, context,
 			"cas_cache_rollback_complete");
-	if (!context->rollback_thread) {
+	if (IS_ERR(context->rollback_thread)) {
 		kfree(context);
 		module_put(THIS_MODULE);
-		return -ENOMEM;
+		return PTR_ERR(context->rollback_thread);
```

Comment:
You are using `context` after freeing it

Reply:
I read the same email in the morning. You were faster.